documentation/ipi-install/modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -13,18 +13,21 @@ The installer for installer-provisioned {product-title} clusters validates the h
 | 9th Generation | iLO4 | N/A
 
 .2+| Dell | 14th Generation | iDRAC 9 | v4.20.20.20 - 04.39.00.00
+v04.40.00.00 with a workaround.
 
 | 13th Generation .2+| iDRAC 8 | v2.75.75.75+
 
 |====
 
+
+
 [NOTE]
 ====
 Refer to the hardware documentation for the nodes or contact the hardware vendor for information on updating the firmware.
 
 There are no known firmware limitations for HP servers.
 
-For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: **Configuration->Virtual Media->Attach Mode->AutoAttach**. There is a known issue with iDRAC 9 firmware version 04.40.00.00.
+For Dell servers, ensure the {product-title} cluster nodes have AutoAttach Enabled through the iDRAC console. The menu path is: **Configuration->Virtual Media->Attach Mode->AutoAttach**. There is a known issue with iDRAC 9 firmware version 04.40.00.00 which has a workaround when setting the console to use HTML5 instead of the default setting.
 ====
```

Comment:
I'm fine with the change, just wondering if we need to tell the path to change the console type (like we have in the AutoAttach)

Reply:
If you have it at hand I can add  to the PR to be validated by @johnwilkins

Reply:
1- login in the idrac page
2- access virtual console settings
3- On Virtual Console tab change the `Plug-in Type` to `HTML5`

Reply:
I sent in another upstream PR already on this. I will add the menu path to the downstream PR so we can get it in before COB today.